data/persons.txt

```diff
@@ -1850,3 +1849,174 @@ ship "Modified Ladybug"
+
+			"Looks good to me."
+			"This should be const"
+			"Is this the best name?"
+			"<reviewer> requested a review from (Amazinite/Tehhowch)"
+			"The standard copyright header was changed recently."
+
+			"Please use the template."
+			"The decision to implement fancy quote is one of the worst in computer history."
+			"	This needs another indent."
+
+	phrase
+		word
+			"Issue #"
+
+		word
+			"4034"
+			"2979"
+			"645"
+			"4124"
+			"1255"
+			"5311"
+			"4460"
+			"651"
+			"3148"
+			"802"
+			"3907"
+			"362"
+
+#		EjoThims
+	phrase
+		word
+			"If you do not want your pull request to be merged, "
+			"If you prefer your pull request sits idle, "
+			"If you want your pull request to be closed, "
+		word
+			"make sure you keep adding content to it"
+			"always change unrelated things"
+			"do not ever test it, and don't add integration tests, whatever that is"
+			"try not to respect existing syntax and conventions"
+			"just forget about it for a while"
+			"use the forbidden word: 'realism'"
+			"keep the new feature with the data changes"
+			"do not, under any circumstances, be willing to compromise"
+			"make sure to ignore what the reviewers tell you to do"
+			"don't write an issue about it first"
+		word
+			"!" 5
+			"." 5
+			". Trust me, it works."
+			". You shouldn't do those things though, I'm just kidding."
+
+	ship "Waverider" "Pain of the Proofreader"
+		"never disabled"
+	ship "Waverider" "Regret of the Art Critics"
+		"never disabled"
+	ship "Waverider" "Curse of the Code Herders"
+		"never disabled"
+	ship "Waverider" "Tears of the Choreographers"
+		"never disabled"
+	ship "Waverider" "Blood of the Adjuncators"
+		"never disabled"
+	ship "Waverider" "Madness of the Balancers"
+		"never disabled"
+
+ship "Waverider"
+	sprite "ship/waverider"
+	thumbnail "thumbnail/waverider"
+	attributes
+		category "Light Warship"
+		"cost" 0
+		"shields" 10000
+		"hull" 10000
+		"required crew" 1
+		"bunks" 1
+		"mass" 150
+		"drag" 1.5
+		"heat dissipation" 1
+		"fuel capacity" 1000
+		"cargo space" 10000
+		"outfit space" 1000
+		"weapon capacity" 1000
+		"engine capacity" 1000
+		weapon
+			"blast radius" 340
+			"shield damage" 3400
+			"hull damage" 1700
+			"hit force" 5000
+	outfits
+		"Particle Cannon" 4
+		"S-970 Regenerator"
+		"Dwarf Core"
+		"LP072a Battery Pack"
+		"Liquid Nitrogen Cooler"
+		"Catalytic Ramscoop"
+		"Tactical Scanner"
+		"Cargo Scanner"
+		"Outfit Scanner"
+		"Asteroid Scanner"
+
+		"A120 Atomic Thruster"
+		"A125 Atomic Steering"
+		"A125 Atomic Steering"
+		"Scram Drive"
+
+#		"Emotional Baggage"
+	
+	gun -22.5 -56.5 "Particle Cannon"
+	gun 22.5 -56.5 "Particle Cannon"
+	gun -31.5 -7 "Particle Cannon"
+	gun 31.5 -7 "Particle Cannon"
+	engine 0 56
+	engine -23 64
+	engine 23 64
+	description `The Waverider was a ship that was first requested and never used for Five Years, Seven Months and Twenty-Seven Days.`
+	description `	Its disappointment was so immense, it willed itself into being.`
```

Comment:
```suggestion
	engine 0 56
	engine -23 64
	engine 23 64
	gun -22.5 -56.5 "Particle Cannon"
	gun 22.5 -56.5 "Particle Cannon"
	gun -31.5 -7 "Particle Cannon"
	gun 31.5 -7 "Particle Cannon"
	description `The Waverider was a ship that was first requested and never used for five years, seven months and twenty-seven days.`
	description `	Its disappointment was so immense, it willed itself into being.`
```
Engine definitions go before weapons. Also, the caps aren't really necessary to get the point across when you already are explaining a long period of time with several words.

Reply:
Noted.